File: Frontend-reactjs/src/service/ProfitService.js
import { axiosInstance } from "./Service";

class ProfitService{
    addProfit(profit){
        return axiosInstance.post('/addProfit',profit);
    }

    async getProfit() {
        const response = await axiosInstance.get('/getProfit')
        return response;
    }
}

export default new ProfitService()

File: Backend/src/main/java/com/example/SimpleWebApp/service/ProfitService.java
package com.example.SimpleWebApp.service;

import com.example.SimpleWebApp.entity.Profit;
import com.example.SimpleWebApp.repository.ProfitRepository;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.stereotype.Service;

import java.util.List;

@Service
public class ProfitService {

    @Autowired
    private ProfitRepository profitRepository;

    public Profit saveProfit(Profit profit){
        return profitRepository.save(profit);
    }

    public List<Profit> getProfit(){
        return profitRepository.findAll();
    }
}
